pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -9613,6 +9619,23 @@
   hoc_ai_101_for_educators_desc: "Discover the groundbreaking world of artificial intelligence (AI) and its transformative potential in education with our foundational online learning series for teachers."
   hoc_ai_impacts_desc: "Engage your classroom with  teacher-led lessons that explore the impacts of artificial intelligence and invite learners to think of solutions to some of AI's issues."
   hoc_ai_activities_desc: "These all-new Hour of Code activities introduce users to the world of artificial intelligence in a fun, engaging way."
+  hoc_ai_additional_educator_resources_desc: "There is so much to learn about how AI can support you and your students. Check out these courses and tools from partners."
+  hoc_ai_resources_adobe_title: "Bring generative AI into your classroom with Adobe Express"
+  hoc_ai_resources_adobe_desc: "In this self-paced Lightning Learning course, you'll learn about the many applications Adobe Express's new generative AI features have for teaching and learning. You'll also explore creative and simple activities that support your students throughout any project."
+  hoc_ai_resources_adobe_aria_label: "Explore the Adobe Express course"
+  hoc_ai_resources_day_of_ai_title: "Day of AI"
+  hoc_ai_resources_day_of_ai_desc: "Join a global community to bring a Day of AI to your school. Explore engaging activities and teacher resources for all grade levels that help build AI literacy and a foundational knowledge of artificial intelligence."
+  hoc_ai_resources_microsoft_01_title: "Explore the potential of artificial intelligence"
+  hoc_ai_resources_microsoft_01_desc: "Navigate AI in education by looking at essential AI concepts, techniques, and tools, highlighting practical applications."
+  hoc_ai_resources_microsoft_01_aria_label: "See details for the explore the potential of artificial intelligence lessons"
```

Comment:
Nit: Should we add quotes or capitalization or something so it's clearer that "Explore the potential of AI lessons" is the title, because without it and without any punctuation it's a confusing sentence

Reply:
Since this is for screen readers I don't think punctuation will make a difference — I'll change it to `See details for exploring the potential of artificial intelligence lessons`